app/src/main/res/values/strings.xml

```diff
@@ -397,7 +397,8 @@
     <string name="calling_label_speaker">Speaker</string>
     <string name="calling_label_flip">Flip</string>
     <string name="calling_label_ringing_call">Ringing...</string>
-    <string name="calling_label_incoming_call">Calling..</string>
+    <string name="calling_label_incoming_call">Calling...</string>
+    <string name="calling_label_incoming_call_someone_calling">%s is calling...</string>
```

Comment:
```suggestion
    <string name="calling_label_incoming_call">Calling…</string>
    <string name="calling_label_incoming_call_someone_calling">%s is calling…</string>
```

Instead of 3 dots, we can use the [Horizontal Ellipsis](https://www.compart.com/en/unicode/U+2026) character. 
This prevents the ellipsis to break in two lines when reaching the end of the screen for example.

For example:
```
Vitor Schwaab is calling..
.
```

Would become:
```
Vitor Schwaab is calling…
```

Or, worst case scenario (if it can't fit into a single line):
```
Vitor Schwaab is 
calling…
```